easy/Challange3.js

```diff
@@ -0,0 +1,13 @@
+// Given two numbers, return true if the sum of both numbers is less than 100. Otherwise return false.
+
+function addition(num1,num2)
+{
+    let result=num1+num2;
```

Comment:
Prefer `const` over `let` if value of variable is not going to change

Reply:
Changed the variable to const over let
commit Id c80b5b574488ea8969fae231fe85bc7f896982a6